recipes/napari-image-stacker/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "napari-image-stacker" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-image-stacker-{{ version }}.tar.gz
+  sha256: 658d9d176d44701c66a8a04b22f4f1759d98ffc8c7bbbabc63d442f8e3c6e14c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - napari
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - python >=3.7
+    - regex
```

Comment:
I don't think this is needed. I took a look and you are only [importing `re`](https://github.com/RobAnKo/napari-image-stacker/blob/main/src/napari_image_stacker/_dock_widget.py#L20), which is part of the standard library :/ 

```suggestion
```

Reply:
@RobAnKo - I think you can fix that in your setup.cfg too for the next release?

Reply:
Ah, thanks for the suggestion, I wasn't aware that re is part of the standard library. I'll fix it right away.

Is it sufficient to change it in my setup.cfg and the change transfers to recipe/napari-image-stacker/meta.yaml automatically? If no, how do I fix it in meta.yaml? Or do you fix it?

I'm brand new to conda-forge and the concept of recipes, so excuse my ignorance :see_no_evil:

Reply:
We took care of that here! Just change your repo so the next PyPI release takes into consideration. When that happens we need to remove `regex` from `test.requires` because that workaround won't be needed, but that's it!

Thank you!